css/main.css

```diff
@@ -0,0 +1,620 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'COCOGOOSE', sans-serif;
+  letter-spacing: 0.038rem;
+}
+
+:root {
+  --black: #272a31;
+  --orange: #ec5242;
+  --gray: #d3d3d3;
+}
+
+.container {
+  padding-left: 5%;
+  padding-right: 5%;
+}
+
+.landing-page,
+.contact-us {
+  height: auto;
+  width: 100%;
+  background-color: rgba(211, 211, 211, 0.4);
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-top: 10%;
+  padding-bottom: 10%;
+}
+
+.landing-page p,
+.contact-us p {
+  margin-top: 20px;
+}
+
+.landing-page::after,
+.contact-us::after {
+  content: '';
+  background-image: url(../images/homeBackground.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  opacity: 0.3;
+}
+
+.landing-page img,
+.contact-us img {
+  width: 4%;
+}
+
+.logo-info p,
+.past-job-fair > p {
+  color: var(--black);
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  font-weight: 600;
+  text-align: center;
+}
+
+.logo-info p {
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
+
+.past-job-fair > p {
```

Comment:
as per the design requirement, you should similar font size, padding, and shadows as the original design

your design 
![image](https://user-images.githubusercontent.com/60612329/181997033-f3368924-7f6b-4bb3-be2f-26cbb7abf05e.png)

required design
![image](https://user-images.githubusercontent.com/60612329/181997045-0632c8fa-94c6-4e21-9bd0-6145d30d5ff1.png)

Reply:
@alexander16108 mmm you may be right but there are no specific details about that it is a photo and you make these font sizes just by sense besides I try to make the size bigger to make it easy to read as it is a good practice . if you tried to change the design width you will notice I change the size according to the device width.